### DISCUSSION TITLE
一起很困扰，今天找到问题了。

### DISCUSSION THREAD

**Discussion by kevzcr** [shuax/MouseInc.Settings#421] [General]
![image](https://user-images.githubusercontent.com/88956040/191147522-26d38dbb-7ab5-4339-a47f-558714ce53b5.png)

![image](https://user-images.githubusercontent.com/88956040/191148472-c401ef44-cac3-42d2-861c-2c3f3345cc00.png)

**Comment by shuax** [comment]
你的离线包版本不对吧

**Reply by kevzcr** [reply]
> 你的离线包版本不对吧
更新了`MouseInc.Settings.zip` 问题完美解决。
